### PR TITLE
Add Quota Command's for limits and history

### PR DESCRIPTION
**Pull request by apoclyps**
### What's Changed

Adds commands for quota limits and history (with oss usage available with the -oss flag) for an organisation. 

##### Example Usage

```bash
cloudsmith quota limits example-org

cloudsmith quota limits example-org -oss
```

Screenshot 2020-09-17 at 15 46 05

```bash
cloudsmith quota history example-org

cloudsmith quota history example-org -oss
```

Screenshot 2020-09-17 at 15 46 30

### CODE REVIEW

**Comment by lskillen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cloudsmith_cli/cli/commands/quota/command.py  1
- cloudsmith_cli/cli/commands/quota/history.py  4
- cloudsmith_cli/cli/commands/quota/quota.py  4
- cloudsmith_cli/core/api/quota.py  12
         

Clones added
============
- cloudsmith_cli/cli/commands/quota/history.py  3
- cloudsmith_cli/cli/commands/quota/quota.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Cloudsmith/cloudsmith-cli/pullRequest?prid=6196275&bid=20213952)

File: cloudsmith_cli/core/api/quota.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+def quota_limits(owner=None, oss=False, **kwargs):
+    """Get Quota for namespace"""
+    client = get_quota_api()
+
+    api_kwargs = {}
+    api_kwargs.update(
+        {param: value for param, value in kwargs.items() if value is not None}
+    )
+
+    if owner and oss:
```
Comment (lskillen) [inline cloudsmith_cli/core/api/quota.py:26]: Can `owner` be empty/None? Does the validator not prevent this?

Comment (apoclyps) [reply cloudsmith_cli/core/api/quota.py]: 👍  removed

File: cloudsmith_cli/cli/commands/quota/quota.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            click.style(str(bandwidth.get("used", "")), fg="white"),
+            click.style(str(bandwidth.get("configured", "")), fg="white"),
+            click.style(str(bandwidth.get("plan_limit", "")), fg="white"),
+            click.style(str(bandwidth.get("percentage_used", "")), fg="white"),
+        ]
+    )
+    click.echo()
+    utils.pretty_print_table(headers, rows)
+
+    click.echo(click.style("\nStorage Quota", bold=True, fg="white"))
```
[the remark this answers is not in the document]
Comment (apoclyps) [reply cloudsmith_cli/cli/commands/quota/quota.py]: replaced with `click.echo()`.

File: cloudsmith_cli/core/api/quota.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+def quota_history(owner=None, oss=False, **kwargs):
+    """Get Quota history for namespace"""
+    client = get_quota_api()
+
+    api_kwargs = {}
+    api_kwargs.update(
+        {param: value for param, value in kwargs.items() if value is not None}
+    )
+
+    if owner and oss:
```
[the remark this answers is not in the document]
Comment (apoclyps) [reply cloudsmith_cli/core/api/quota.py]: flipped, and removed owner - it should always be set. this was leftover from experimenting before the API bindings were ready.

**Comment by lskillen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cloudsmith_cli/cli/commands/quota/quota.py  4
- cloudsmith_cli/cli/commands/quota/history.py  4
- cloudsmith_cli/cli/commands/quota/command.py  1
- cloudsmith_cli/core/api/quota.py  10
         

Clones added
============
- cloudsmith_cli/cli/commands/quota/quota.py  3
- cloudsmith_cli/cli/commands/quota/history.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Cloudsmith/cloudsmith-cli/pullRequest?prid=6196275&bid=20213952)

**Review by lskillen — APPROVED** [review]
❤️ 

`pretty_print_table` ftw.

**Pull request merged**